WooCommerce/Classes/ViewRelated/Products/Edit Product/Edit Price/ProductEditPriceSettingsViewController.swift

```diff
@@ -0,0 +1,417 @@
+import UIKit
+import Yosemite
+
+// MARK: - ProductEditPriceSettingsViewController
+//
+final class ProductEditPriceSettingsViewController: UIViewController {
```

Comment:
nit: what do you think about the name `ProductEditPriceSettingsViewController` vs. `ProductPriceSettingsViewController`, with or without `Edit` in the class name? I slightly prefer not having `Edit` in the settings screen class name, as `Settings` sort of implies the `Edit`/`Configure` action. I'm fine either way though, if we go with `Edit` I'll update the names for the shipping and inventory view controllers 🙂

Reply:
Make totally sense! I'm going to change it 😉